docs/rucio.md

```diff
@@ -145,25 +145,25 @@ By typing in the terminal:
 ```console
 $ rucio whoami
 ```
-You should see your username being recognised. If it is your first time using tokens, you will be redirected to a link starting with  'https://escape-rucio-auth.cern.ch/auth/...', click on it and choose the duration of your token. You should be all set up to run your rucio commands!
+you should see your username being recognised. If it is your first time using tokens, you will be redirected to a link starting with  'https://escape-rucio-auth.cern.ch/auth/...'; click on it and choose the duration of your token. You should be all set up to run your rucio commands!
 
 ## 2. Docker image installation 
 
 Docker needs to be installed following the [Docker installation](https://docs.docker.com/get-docker/) instructions. The procedure will change depending on your operating system. 
-The Docker file will extend the Rucio image and will enable the user to interact with the Data Lake. Further information can be found [here](https://github.com/cern-vre/containers/tree/main/rucio-client). 
+The Docker file will extend the Rucio image and will enable the user to interact with the Data Lake. Further information can be found [here](https://github.com/vre-hub/vre/blob/main/containers/rucio-client/Dockerfile). 
```

Comment:
Could you please point to the parent directory @AndresTanasijczuk ? `.../containers/rucio-client` rather than to the `Dockerfile`.

Reply:
Done.